src/main/java/com/microsoft/azure/helium/config/BuildConfig.java

```diff
@@ -12,10 +18,26 @@
 
     public String getBuildVersion(){
 
-        String buildName = buildProperties.getName();
-        String buildVersion = buildProperties.getVersion();
-        String buildTime =  String.valueOf(buildProperties.getTime().getEpochSecond());
-        System.out.println("buildName " + buildName +"_"+ "buildVersion "+ buildVersion + "_" + "buildTime "+ buildTime );
-        return buildVersion+"."+buildTime ;
+        Instant buildTime = buildProperties.getTime();
+
+        DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+                .withZone(ZoneId.systemDefault());
+
+        System.out.println(DATE_TIME_FORMATTER.format(buildTime));
+
+        //major.minor.MMdd.hhmm
+        String major = buildProperties.getVersion();
+        DateTimeFormatter DATE_TIME_FORMATTER1 = DateTimeFormatter.ofPattern("MMDD")
+                .withZone(ZoneId.systemDefault());
+        String MMDD = DATE_TIME_FORMATTER1.format(buildTime);
+
+        DateTimeFormatter DATE_TIME_FORMATTER2 = DateTimeFormatter.ofPattern("hhmm")
+                .withZone(ZoneId.systemDefault());
+        String hhmm = DATE_TIME_FORMATTER2.format(buildTime);
+
+        String versionNo = major + "." + MMDD + "." + hhmm;
+        System.out.println(versionNo);
```

Comment:
are the system.out.println calls going to "pollute" your logs? Are they necessary?

Reply:
Cleaned!